pkg/local/cedarcoremock/system_maintainer_information.go

```diff
@@ -10,144 +10,224 @@ import (
-		OmDocumentationOnDemand:    false,
-		PlansToRetireReplace:       zero.StringFrom("No"),
-		QuarterToRetireReplace:     zero.StringFrom("2"),
-		RecordsManagementBucket:    []zero.String{zero.StringFrom("Record 1"), zero.StringFrom("Record 2")},
-		SourceCodeOnDemand:         false,
-		SystemCustomization:        zero.StringFrom("50% customization"),
-		SystemDesignOnDemand:       false,
-		SystemProductionDate:       zero.TimeFrom(time.Date(2025, 1, 1, 0, 0, 0, 0, time.UTC)),
-		SystemRequirementsOnDemand: false,
-		TestPlanOnDemand:           false,
-		TestReportsOnDemand:        false,
-		TestScriptsOnDemand:        false,
-		YearToRetireReplace:        zero.StringFrom("2025"),
+		AdHocAgileDeploymentFrequency:         zero.StringFrom("Daily"),
+		AgileUsed:                             false,
+		AuthoritativeDatasource:               zero.StringFrom("Wikipedia"),
+		BusinessArtifactsOnDemand:             true,
+		DataAtRestEncryptionKeyManagement:     zero.StringFrom("Encryption"),
+		DeploymentFrequency:                   zero.StringFrom("Quarterly"),
+		DevCompletionPercent:                  zero.StringFrom("75%"),
+		DevWorkDescription:                    zero.StringFrom("Testing in production"),
+		EcapParticipation:                     false,
+		FrontendAccessType:                    zero.StringFrom("IPv4"),
+		HardCodedIPAddress:                    false,
+		IP6EnabledAssetPercent:                zero.StringFrom("0%"),
+		IP6TransitionPlan:                     zero.StringFrom("This system will not be transitioned to IPv6"),
+		IPEnabledAssetCount:                   0,
+		LegalHoldCaseName:                     zero.StringFrom("Case hold"),
+		LocallyStoredUserInformation:          false,
+		MajorRefreshDate:                      zero.TimeFrom(time.Date(2025, 2, 2, 0, 0, 0, 0, time.UTC)),
+		MultifactorAuthenticationMethod:       []zero.String{zero.StringFrom("Google Authenticator"), zero.StringFrom("Microsoft Authenticator")},
+		MultifactorAuthenticationMethodOther:  zero.StringFrom("Other mfa methods"),
+		NetAccessibility:                      zero.StringFrom("Accessible to public internet"),
+		NetworkTrafficEncryptionKeyManagement: zero.StringFrom("Encryption"),
+		NoMajorRefresh:                        false,
+		NoPersistentRecordsFlag:               false,
+		NoPlannedMajorRefresh:                 false,
+		OmDocumentationOnDemand:               false,
+		PlansToRetireReplace:                  zero.StringFrom("No"),
+		QuarterToRetireReplace:                zero.StringFrom("2"),
+		RecordsManagementBucket:               []zero.String{zero.StringFrom("Record 1"), zero.StringFrom("Record 2")},
+		RecordsManagementDisposalLocation:     zero.StringFrom("Recycle bin"),
+		RecordsManagementDisposalPlan:         zero.StringFrom("Weekly deletion"),
+		RecordsUnderLegalHold:                 false,
+		SourceCodeOnDemand:                    false,
+		SystemCustomization:                   zero.StringFrom("50%% customization"),
```

Comment:
Was this extra `%` on purpose?

Reply:
it was, because I looked at this and was thinking it was a format string - i am wrong! Thanks Clay